src/main/java/bio/terra/cli/command/Status.java

```diff
@@ -43,7 +43,9 @@ private void printText(StatusReturnValue returnValue) {
     if (workspaceContext.isEmpty()) {
       OUT.println("There is no current Terra workspace defined.");
     } else {
-      OUT.println("Terra workspace: " + workspaceContext.getWorkspaceId());
+      OUT.println("Terra workspace id: " + workspaceContext.getWorkspaceId());
+      OUT.println("Display name: " + workspaceContext.getWorkspaceDisplayName());
```

Comment:
nit: maybe put the workspace display name before the workspace id. Consider also just `name` instead of display name.

Reply:
I'll pass on this one for now.